### ISSUE TITLE
[Bug]: access violation after fix

### ISSUE THREAD

**Issue by Deadlineem** [Yimura/YimMenu#2731]
### Describe the bug

R* updated, you guys suggested the fix, i compiled it with the new PD and it works, i have not crashed but console logs this
[cout.log](https://github.com/YimMenu/YimMenu/files/14349385/cout.log)

### Steps To Reproduce

compile yim x64-release with the PD fix:
"PD"
"48 8D 05 ? ? ? ? 48 8B F9 48 89 01 48 83 C1 08 E8 ? ? ? ? 33 C0"

Load/Inject and just play, console should log

### Expected Behavior

Unsure, no access violations id assume, still works without crashing so far even with the error

### Logs and/or screenshots

see cout.log

### Store

Epic Games

### Language

English

### YimMenu Language

English (US)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by Yimura** [comment]
According to the log file you were using a dirty branch, you'll have to use the DLL that the Github action bot builds before we can actually take a look at the underlying issue. This is required from issue reports to prevent us from chasing a bug that doesn't exist on an actual build of YimMenu.

```
[17:16:31.1492298][INFO/main.cpp:160] Git Info
	Branch:	master
	Hash:	4e294162fbed6bc5674b4795b8d133092badc0aa-dirty
	Date:	Mon Feb 19 05:37:00 2024
```

Notice the `-dirty` at the end of the hash, which is based of off 4e294162fbed6bc5674b4795b8d133092badc0aa. This makes it impossible for us to properly diagnose the issue by us because that commit hash doesn't have the fixes implemented by #2727. 
And we can only guess what exactly you've modified besides the fix mentioned.

**Comment by xiaoxiao921** [comment]
Report when you experience actual issues, we couldnt care less about random game exceptions

**Comment by Yimura** [comment]
Replicate the bug with the latest version of YimMenu and then we can actually look into fixing it, an extra step for you to check before submitting an issue is to replicate the bug without having any Lua's loaded as they may also be modifying memory in ways we cannot predict.

Edit:
You can post here and we can reopen the issue if it seems legit and actually replica-table.

**Comment by Deadlineem** [comment]
> According to the log file you were using a dirty branch, you'll have to use the DLL that the Github action bot builds before we can actually take a look at the underlying issue. This is required from issue reports to prevent us from chasing a bug that doesn't exist on an actual build of YimMenu.
> 
> ```
> [17:16:31.1492298][INFO/main.cpp:160] Git Info
> 	Branch:	master
> 	Hash:	4e294162fbed6bc5674b4795b8d133092badc0aa-dirty
> 	Date:	Mon Feb 19 05:37:00 2024
> ```
> 
> Notice the `-dirty` at the end of the hash, which is based of off [4e29416](https://github.com/YimMenu/YimMenu/commit/4e294162fbed6bc5674b4795b8d133092badc0aa). This makes it impossible for us to properly diagnose the issue by us because that commit hash doesn't have the fixes implemented by #2727. And we can only guess what exactly you've modified besides the fix mentioned.

i cloned a fresh copy of yims repo and added the fix info gir posted from  https://github.com/YimMenu/YimMenu/pull/2727
but yeah, my lua script does bend alot  of rules, ill use this for now till you guys push the update then try with the official dll and see if the issue replicates/persists.

**Comment by Yimura** [comment]
> i cloned a fresh copy of yims repo and added the fix info gir posted from #2727 but yeah, my lua script does bend alot of rules, ill use this for now till you guys push the update then try with the official dll and see if the issue replicates/persists.

The fix has been merged and a new DLL has been built already.
